### DISCUSSION TITLE
gateway api on gke got error "Size of the url map exceeds the maximum permitted size"

### DISCUSSION THREAD

**Discussion by pochingliu131** [kubernetes-sigs/gateway-api#3089] [Q&A]
Hello everyone. 

Recently I started migrating my services from istio to gateway api on my multiple cluster gke. Because I have 100+ HTTPRoute need to create. When I already declared 30+ HTTPRoute I encounter some error "Size of the url map exceeds the maximum permitted size" It seems like that I hit "Size of URL maps" limit on my gcp(only 64KB for external LB) but it can't be increased or changed on myself. I understand it's all gcp's fault not gateway api. But maybe someone can help me on here? 

Reference : 
https://cloud.google.com/load-balancing/docs/quotas#url_maps

**Comment by robscott** [comment]
Hey @totoroliu0131, do you mind filing a bug through GCP support so we can follow up there with all your project information?

**Reply by pochingliu131** [reply]
yeap, I already sent request form to GCP, I understood that is not gateway api problem, just curious am I the only one experiencing this problem?
